units/crons.go

```diff
@@ -846,19 +851,25 @@ func PopulateHostCreationJobs(env evergreen.Environment, part int) amboy.QueueOp
 			if h.UserHost || h.SpawnOptions.SpawnedByTask {
 				// pass:
 				//    always start spawn hosts asap
+
 			} else {
-				if submitted > env.Settings().HostInit.HostThrottle {
-					// throttle hosts, so that we're starting very
-					// few hosts on every pass. Hostinit runs very
-					// frequently, lets not start too many all at
-					// once.
-					continue
+				num, _ := distroCount[h.Distro.Id]
```

Comment:
I think the _ is extra if we aren't checking whether the key is present.
https://play.golang.org/p/wlKk8YLmzCo